package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "modmail-userinfo",
-  "version": "0.0.0",
+  "version": "1.6.2",
```

Comment:
This isn't needed, and I'd rather avoid the headache of having to remember to maintain the version in package.json. You only need to set version numbers in these if you're going to publish to a package registry.